.github/workflows/release-notification.yml

```diff
@@ -0,0 +1,39 @@
+name: Release notification
+on:
+  release:
+    types:
+      - published
+
+env:
+  RELEASE_NAME: ${{ github.event.release.name }}
+
+jobs:
+  send-sms:
+    if: ${{ startsWith('dialtone/v', env.RELEASE_NAME) }}
+    name: Send SMS
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    env:
+      API_URL: https://us-central1-dp-dialtone-design-system.cloudfunctions.net/send-sms-to-dialtone-test-channel
```

Comment:
is this supposed to be locked to a location like this "us-central1"? No location agnostic url? If expected it's fine.

Reply:
Yes, it's expected. It is something specific of cloud functions version 2

Reply:
k np looks good then